tests/appsec/test_reports.py

```diff
@@ -36,6 +36,11 @@
 class Test_StatusCode(BaseTestCase):
     """ Appsec reports good status code """
 
+    @irrelevant(
+        library="java",
+        weblog_variant="spring-boot-openliberty",
+        reason="This is a bug but @bug annotation fails, https://datadoghq.atlassian.net/browse/APPSEC-6583",
+    )
     def test_basic(self):
         r = self.weblog_get("/path_that_doesn't_exists/", headers={"User-Agent": "Arachni/v1"})
         assert r.status_code == 404
```

Comment:
can you retry with the bug decorator,  replace this line with, and retry ? 

```suggestion
        interfaces.library.add_assertion(r.status_code == 404)
```

Reply:
you did 

```
assert  interfaces.library.add_assertion(r.status_code == 404)
```

where it should be

```
  interfaces.library.add_assertion(r.status_code == 404)
```

;)

Reply:
:scream: I'm so sorry